### PR TITLE
Feature/setup check for running containers

### PR DESCRIPTION
**Pull request by Johanndutoit**
Tested and happy with the result, killed a few VM's and ran the setup.sh script again which made sure they will all started again.

Tested at a few stages during the install just to be sure.

Looks good, @bruskiza just to review if the logic makes sense.

### CODE REVIEW

**Comment by Johanndutoit** [comment]
@jonathanendersby ^ more eyes can't hurt :)

**Comment by bruskiza** [comment]
LGTM

File: scripts/setup.sh [hunk 1]
```diff
@@ -185,6 +185,41 @@ if [ ! -f /var/goddard/setup.lock ]; then
...
+			echo "Sanity Check if $tkey is actually running"
+
+			# done
+			echo "{\"build\":\"busy\",\"process\":\"Making sure $tkey is actually running\",\"timestamp\":\"$( date +%s )\"}"  > /var/goddard/build.json
+
+			# post to server
+			curl -X POST -d @/var/goddard/build.json http://hub.goddard.unicore.io/report.json?uid=$(cat /var/goddard/node.json | jq -r '.uid') --header "Content-Type:application/json"
+
+			# get the running apps
+			running_app_container=$(docker ps | grep $tkey) || true
```
Comment (bruskiza) [inline scripts/setup.sh:218]: @Johanndutoit  post this back to the hub server so we have a record of the "before" state.

**Comment by jonathanendersby** [comment]
+1 Gravy

File: scripts/setup.sh [hunk 2]
```diff
@@ -185,6 +185,57 @@ if [ ! -f /var/goddard/setup.lock ]; then
...
+
+		# update in file
+		running_docker_status=$(docker ps)
+
+		# clean up the docker output
+		cleaned_docker_status=$(echo $running_docker_status | sed -r 's/[\"]+/\\\"/g')
+		# cleaned_docker_status=$(python2 -c 'import sys, urllib; print urllib.quote(sys.argv[1])' "$running_docker_status")
+
+		# done
+		echo "{\"build\":\"busy\",\"process\":\"Output from Docker: ${cleaned_docker_status}\",\"timestamp\":\"$( date +%s )\"}"  > /var/goddard/build.json
```
Comment (bruskiza) [inline scripts/setup.sh:199]: @jonathanendersby it's here for that ticket... i don't see any messages today though.

**Pull request merged**